### PR TITLE
Blog post for 17 Dec governance mtg

### PR DESCRIPTION
**Pull request by valgrimm**
This is the blog post to summarize the 17th December governance meeting.

### CODE REVIEW

**Comment by JohnTitor** [comment]
I think you should put it in `inside-rust` dir and use file format like this: https://github.com/rust-lang/blog.rust-lang.org/blob/master/posts/inside-rust/2019-12-10-governance-wg-meeting.md

**Comment by valgrimm** [comment]
Thank you @JohnTitor ! I have corrected the location of the post (and also took the opportunity to fix the structure). Best regards.

File: posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
+
+
+Hello everyone! 
+
+On Tuesday 17 December the working group met and built an outline of an RFC to introduce project groups.
+
+## Project groups
+
+You can see the detailed minutes at the link above, but here is a quick summary of what we would like to do with the RFC:
```
Comment (nikomatsakis) [inline posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md]: we should make a link to the minutes
```suggestion
You can see the [detailed minutes] on the [wg-governance] repository, but here is a quick summary of what we would like to do with the RFC: 
```

File: posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* Introduce the project group concept, creation workflow, and lifecycle
+* Provide guidance for how project groups use RFCs
+
+Goals served by introducing project groups include improved visibility into and follow-through on projects,
+as well as supporting better documentation of the design process.
+
+
+## Next meeting
+
+Our next meeting will be Jan 14. The topic is to be determined, but it is planned to discuss that on January 7th informally.
```
Comment (nikomatsakis) [inline posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md]:
```suggestion
Our next meeting will be Jan 14. The topic is to be determined, but it is planned to discuss that on January 7th on the Rust Discord in the `#wg-governance` channel.
```

File: posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+layout: post
+title: "Governance Working Group Update: Meeting 17 December 2019"
+author: Val Grimm
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
+
+
+Hello everyone! 
+
+On Tuesday 17 December the working group met and built an outline of an RFC to introduce project groups.
```
Comment (nikomatsakis) [inline posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md]:
```suggestion
On Tuesday 17 December the working group met and built an outline of an RFC to introduce project groups. If you would like to watch, a recording is available on [YouTube].
```

File: posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Next meeting
+
+Our next meeting will be Jan 14. The topic is to be determined, but it is planned to discuss that on January 7th informally. 
+
+[wg-governance]: https://github.com/rust-lang/wg-governance/
+[detailed minutes]: https://github.com/rust-lang/wg-governance/blob/master/minutes/2019.12.17.md
+[`rust-dev-tools`]: https://github.com/rust-dev-tools/
+[`youtube`]: https://www.youtube.com/watch?v=q6us8yhDDUU&feature=youtu.be
```
Comment (nikomatsakis) [inline posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md]:
```suggestion
[YouTube]: https://www.youtube.com/watch?v=q6us8yhDDUU&feature=youtu.be
```

File: posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+as well as supporting better documentation of the design process.
+
+
+## Next meeting
+
+Our next meeting will be Jan 14. The topic is to be determined, but it is planned to discuss that on January 7th informally. 
+
+[wg-governance]: https://github.com/rust-lang/wg-governance/
+[detailed minutes]: https://github.com/rust-lang/wg-governance/blob/master/minutes/2019.12.17.md
+[`rust-dev-tools`]: https://github.com/rust-dev-tools/
```
Comment (nikomatsakis) [inline posts/blob/master/posts/inside-rust/2019-12-17-governance-wg-meeting.md]:
```suggestion
```

File: posts/blob/master/posts/inside-rust/2019-12-20-governance-wg-meeting.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+layout: post
```
Comment (nikomatsakis) [inline posts/blob/master/posts/inside-rust/2019-12-20-governance-wg-meeting.md]: whoops wait a minute, @valgrimm, I see that this post was kind of in the wrong place :) I'll patch it...

**Pull request merged**